ecs-agent/api/resource/ebs_discovery_windows.go

```diff
@@ -16,6 +16,108 @@
 
 package resource
 
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+
+	log "github.com/cihub/seelog"
+	"github.com/pkg/errors"
+)
+
+const (
+	volumeDiscoveryTimeoutDuration = 5 * time.Second
```

Comment:
Any issues if we use the `ebsnvmeIDTimeoutDuration` defined in ebs_discovery.go?

Reply:
+1